frontend/src/App.jsx

```diff
@@ -1,19 +1,22 @@
-import React, { useContext } from 'react';
+import React from 'react';
 import './App.css';
 import GamemodeBar from './components/GamemodeBar';
 import P5Wrapper from './components/P5Wrapper';
+import PostGame from './components/PostGame';
 import { Provider } from './context/GamemodeContext';
 
 
 function App() {
 
   return (
-    <Provider>
-      <GamemodeBar />
-      <br></br>
-      <br></br>
-      <P5Wrapper/>
-    </Provider>
+    <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+      <Provider>
+        <PostGame/>
+        <GamemodeBar/>
```

Comment:
I think it's generally best practice to do the conditonal stuff when the child component is called in the parent component. For our game, I guess it's okay since it is an SPA, but in general components should be generalized so they don't use conditions inside them like that

Don't have to change this now since atleast the App.jsx file looks clean, but changing this might also make it easier to understand in App.jsx when certain things are loaded, and also when they are not (rather than going to each components file)